src/index.js

```diff
@@ -6,27 +6,59 @@ document.addEventListener("DOMContentLoaded", () => {
   // Fetcheroo.get("users", getUsersCallback);
   // Fetcheroo.get("credit_cards", getCreditCardsCallback);
 
+
+
+  // PAGES --------------------------------------------------------
+  // The main content container is the "main page"
+  const mainContent = document.querySelector('#main-content');
+  // Call updatePageContent whenever you need to change the "page"
+  const updatePageContent = (innerHTMLContent) => {
+    mainContent.innerHTML = innerHTMLContent;
+  }
```

Comment:
`updatePageContent()` is a new helper function that you should call everytime you change pages